compose.yml

```diff
@@ -11,11 +11,7 @@ services:
     ports:
       - "8002:80"
 
-  nginx3:
-    image: nginx:latest
-    ports:
-      - "8003:80"
-  nginx3:
+
```

Comment:
I am not sure what you wanted to do here.

Reply:
@NtBahati good

Reply:
@NtBahati i was trying to make sure i am using the right base image

Reply:
Me too.